excel/credential/tool.gpt

```diff
@@ -1,6 +1,6 @@
 Name: Microsoft Excel OAuth Credential
 Share Credential: ../../oauth2 as excel.write
-    with GPTSCRIPT_MICROSOFT_EXCEL_TOKEN as env and
+    with GPTSCRIPT_MICROSOFT_EXCEL_TOKEN as token and
```

Comment:
Oh, what was the context to change from env to token?

Reply:
essentially just that `env` isn't quite accurate anymore since it refers to the value used for the oauth token env var specifically, where now we support a map of extra env vars that can be added to this credential as well.